cpumond/cpumond.c

```diff
@@ -170,6 +171,11 @@ int cpumond_loop(cpumond_entry_t *cpumond_entry){
         if (statread(statfd, &total2, &idle2) != 0)
             goto out;
 
+	if (unlikely(total2 == total1)) {
+		sleep(1);
```

Comment:
Don't you need to reread total2?

Reply:
Ahh i see now this is within a loop.

Reply:
Yes, and I used continue with a second sleep as wrapping the update code in a ```if (total2 > total2) {``` still resulted iin Coverity moaning about untrusted divisors :( stupid tool at times.